app/models/reservation.rb

```diff
@@ -0,0 +1,9 @@
+class Reservation < ApplicationRecord
+  belongs_to :user
+  belongs_to :car
+
+  validates :reservation_date, presence: true
```

Comment:
Why don't validate this as date or datetime? This could improve the validations of the model 🐶

Reply:
Fixed as advised. Thanks :relaxed: